docs/concepts/work-pools.md

```diff
@@ -154,6 +156,42 @@ Inspect the work pool:
 
 On success, the command returns the details of the newly created work pool, which can then be used to start agents that poll this pool for work or perform additional configuration of the pool.
 
+#### Base Job Template
+
+Each work pool has a base job template that allows the customization of the behavior of the worker executing flow runs from the work pool. 
+
+Each worker type is configured with a default base job template, making it is easy to get started with a work pool. The default base template defines fields that can be defined either as defaults on the work pool or defined on a deployment.
+
+For example, if we create a `process` work pool via the CLI:
+
+```bash
+$ prefect work-pool create --type process above-ground
```

Comment:
👏

Reply:
10/10 name.

I think we should add `--name` before `above-ground` just to make it more explicit.

Reply:
`name` is an argument and can't be defined with a flag as the CLI is currently implemented. I'll update the text above the command to clarify that `above-ground` is the name of the work pool.